MeshGeoToolsLib/convertMeshNodesToGeometry.cpp

```diff
@@ -0,0 +1,38 @@
+/*
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ */
+
+#include "MeshGeoToolsLib/convertMeshNodesToGeometry.h"
+
+#include "MeshLib/Mesh.h"
+#include "MeshLib/Node.h"
+
+namespace MeshGeoToolsLib
+{
+void convertMeshNodesToGeometry(MeshLib::Mesh const& mesh,
```

Comment:
We already have MeshLib::convertMeshToGeo. It'd probably make sense to either move this functionality there or to move the other functionality here.